src/match/models.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth.models import User
+from django.db import models
+from location.models import Location
+from match.validators import validate_int_list
+from match.validators import validate_times_list
+
+
+class Match(models.Model):
+    status = models.CharField(max_length=20)
+    user_1 = models.ForeignKey(
+        User, default=None, null=True, on_delete=models.CASCADE, related_name="+"
```

Comment:
the related name should be `match` no?

Reply:
or `proposed_match` or `accepted_match` -- it needs to make sense in your context and im not sure what that is